rust/processor/src/schema.rs

```diff
@@ -893,6 +893,45 @@ diesel::table! {
     }
 }
 
+diesel::table! {
```

Comment:
how did these migrations get into the schema? cc @just-in-chang

Reply:
Prob bc @bowenyang007 didn't regenerate the schema after working on tournament

Reply:
Oh Hmm yea you're rigth need to remove